### PR TITLE
Release branch 1.7.1

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
Will push this to master. 

- JDBC version 3.13.14
- jackson-core and jackson-databind 2.12.6 for resolving FasterXML/jackson-databind#3328 
  - Fixes #383

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <groupId>com.snowflake</groupId>
     <artifactId>snowflake-kafka-connector</artifactId>
-    <version>1.7.0</version>
+    <version>1.7.1</version>
```
Comment (sfc-gh-tzhang) [inline pom.xml:15]: I don't see the JDBC version update anywhere, did I miss anything?

Comment (sfc-gh-japatel) [reply pom.xml:15]: I already pushed it to master.

Comment (sfc-gh-japatel) [reply pom.xml:15]: 1ab1a1a9be86c8e429260344c8731dd56b793ee9

**Review by sfc-gh-tzhang — APPROVED** [review]
thanks!

**Pull request merged**